types/vuelidate/vuelidate-tests.ts

```diff
@@ -195,6 +195,16 @@ export class ValidComponent extends Vue {
             return !this.$v.password.minLength
         } else return false
     }
+
+	verifyNoTypeRecursion() {
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/vuelidate/vuelidate-tests.ts;
  Line: 199;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.